etc/profile-a-l/imv.profile

```diff
@@ -0,0 +1,40 @@
+# Persistent local customizations
+include imv.local
+# Persistent global definitions
+include globals.local
+
+include allow-bin-sh.inc
+
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
```

Comment:
```suggestion
```

Reply:
Why remove disable-passwdmgr?

Reply:
@crocket [on Oct 9](https://github.com/netblue30/firejail/pull/4600#discussion_r725546759):

> Why remove disable-passwdmgr?

It does not exist anymore.

See #4461 and the current profile.template.

Reply:
Removed from my local copy.